src/components/MyAccount/ProfileTabs.tsx

```diff
@@ -37,7 +37,9 @@ const ProfileTabs = ({
       content: checkouts ? (
         <CheckoutsTab checkouts={checkouts} patron={patron} />
       ) : (
-        <Text>There was an error accessing your checkouts.</Text>
+        <Text sx={{ marginTop: "s" }}>
```

Comment:
Diego mentioned a preference for the css as prop pattern ie `mt="s"`. Can you update this PR with that?